packages/react-native-worklets/apple/worklets/apple/AnimationFrameQueue.mm

```diff
@@ -39,7 +39,7 @@ - (instancetype)init
   AssertJavaScriptQueue();
   if constexpr (worklets::StaticFeatureFlags::getFlag("IOS_DYNAMIC_FRAMERATE_ENABLED")) {
 #if !TARGET_OS_OSX
-    supportsProMotion = [UIScreen mainScreen].maximumFramesPerSecond > 60;
+    bool supportsProMotion = [UIScreen mainScreen].maximumFramesPerSecond > 60;
```

Comment:
```
bool supportsProMotion = false;
#if !TARGET_OS_OSX
   supportsProMotion = [UIScreen mainScreen].maximumFramesPerSecond > 60;
#endif // !TARGET_OS_OSX
```
I think you should do it like that, because otherwise it will break the build for macOS 🤔